.circleci/config.yml

```diff
@@ -242,21 +242,9 @@ jobs:
     docker:
       - image: circleci/node:10.16.3
     environment:
-      # For multiple
-      NODE_ENV: test
-      # For shared-ganache
-      USE_GANACHE_DEPLOYMENT_CACHE: true
-      GANACHE_CACHE_FOLDER: ../../.ganache-deployments
-      GANACHE_PORT: 8547
-      # For simple-hub
-      FIREBASE_PREFIX: << pipeline.git.branch >>-1
```

Comment:
I think this would need to be also set for `simple-hub`

Reply:
I dont think this integration test uses the hub.